ios/Classes/SwiftTrustdartPlugin.swift

```diff
@@ -122,6 +142,46 @@ public class SwiftTrustdartPlugin: NSObject, FlutterPlugin {
                 }else {
                     result(err.details)
                 }
+            case "getRawPrivateKey":
+                let args = call.arguments as! [String: String]
+                let path: String? = args["path"]
+                let coin: String? = args["coin"]
+                let mnemonic: String? = args["mnemonic"]
+                let passphrase: String? = args["passphrase"]
+        
+                let (isValid, err) = WalletHandler.validate(walletError: WalletError(code: .argumentsNull, message: "[path], [coin], [mnemonic] and [passphrase] are required.", details: nil), path, coin, mnemonic, passphrase)
+                if isValid {
+                    // generate address
+                    let privateKey = WalletHandler().getCoin(coin!).getRawPrivateKey(path: path!, mnemonic: mnemonic!, passphrase: passphrase!)
+                    let (isValid, err) = WalletHandler.validate(walletError: WalletError(code: .addressNull, message: "Failed to generate private key.", details: nil), path, coin, mnemonic, passphrase)
+                    if isValid {
+                        result(privateKey)
+                    }else {
+                        result(err.details)
+                    }
+                }else {
+                    result(err.details)
+                }
+            case "getSeed":
```

Comment:
What is this get seed doing

Reply:
This came from a pull request made by @diopisemou